backend/core/src/main/java/org/sonarsource/sonarlint/core/issue/IssueService.java

```diff
@@ -90,17 +91,19 @@ public class IssueService {
   private final LocalOnlyIssueRepository localOnlyIssueRepository;
   private final ApplicationEventPublisher eventPublisher;
   private final FindingReportingService findingReportingService;
+  private final SeverityModeService modeService;
```

Comment:
Nitpick: `modeService` is also vague, using `severityModeService` would make things more explicit